pyfar/classes/audio.py

```diff
@@ -1,9 +1,12 @@
 """
 The following documents the audio classes and arithmethic operations for
-audio data. More details and background is given in the concepts (
-:py:mod:`audio classes <pyfar._concepts.audio_classes>`,
-:py:mod:`Fourier transform <pyfar._concepts.fft>`,
-:py:mod:`arithmetic operations <pyfar._concepts.arithmetic_operations>`).
+audio data. More details and background is given in the gallery (
+`audio objects <https://pyfar-gallery.readthedocs.io/en/latest/gallery/
+interactive/pyfar_audio_objects.html>`_,
+`Fourier transform <https://pyfar-gallery.readthedocs.io/en/latest/gallery/
+interactive/fast_fourier_transform.html>`_,
+`arithmetic operations <https://pyfar-gallery.readthedocs.io/en/latest/gallery/
+interactive/pyfar_arithmetics.html>`__).
```

Comment:
```suggestion
:doc:`gallery:gallery/interactive/pyfar_audio_objects`, :doc:`gallery:gallery/interactive/fast_fourier_transform`, :doc:`gallery:gallery/interactive/pyfar_arithmetics`).
```
see above to make it work